docs/11-dev-guide.md

```diff
@@ -39,19 +39,18 @@ git checkout --track origin/main
 ```
 
 If you're working with the Sleeper CLI, you can use `sleeper builder` to get a shell inside a Docker container with
-the dependencies pre-installed. You'll need to clone the Git repository, and this will be persisted between executions
-of `sleeper builder`. Use the commands below:
+the dependencies pre-installed. **Note that due to a bug in the GitHub container repository you need to build the
+builder image from scratch**, so it's much easer to use Nix. If you do want to build the builder image from scratch, see the [Sleeper CLI](11-dev-guide.md#sleeper-cli) section below
```

Comment:
This seems not quite correct. It's explained in the getting started guide, so maybe this isn't necessary here?